frontend/src/components/data-provider/Hero.tsx

```diff
@@ -15,7 +16,7 @@ export default function Hero() {
               datasets.
             </p>
             <Button className="mb-16 bg-[#006064] px-6 py-3.5">
```

Comment:
have to set asChild=true to the <Button> component so it renders button properties on its first child, which is the <Link>